sdk/communication/Azure.Communication.Identity/tests/Infrastructure/CommunicationIdentityClientRecordedTestSanitizer.cs

```diff
@@ -6,6 +6,11 @@ namespace Azure.Communication.Pipeline
     public class CommunicationIdentityClientRecordedTestSanitizer : CommunicationRecordedTestSanitizer
     {
         public CommunicationIdentityClientRecordedTestSanitizer()
-            => JsonPathSanitizers.Add("$..token");
+        {
+            JsonPathSanitizers.Add("$..token");
+            JsonPathSanitizers.Add("$..urls");
```

Comment:
does url contain PII information? if not, we do not need to sanitize it. Same for username, is it just an id?

Reply:
No, it's not PII. It's just a static url (common for every client). username is just a random id it looks like, yes.

Reply:
Updated.